pkg/components/scheduler.go

```diff
@@ -283,10 +283,18 @@ func (s *scheduler) createInitUserScript() string {
 	return strings.Join(script, "\n")
 }
 
-func (s *scheduler) prepareInitOperationArchive() {
+func (s *scheduler) prepareInitOperationsArchive() {
+	initOpArchivePath = "/usr/bin/init_operations_archive"
+	// COMPAT(omgronny): drop once new srcipt name is everywhere
+	if _, err := os.Stat(initOpArchivePath); err != nil {
```

Comment:
What is the correct way to get the valid name then? I far as I know we can't just change it from `init_operation_archive` to `init_operations_archive` here

Reply:
If you explain a little what change in ytsaurus you are trying to support in operator it would be easier to decide what to do, but I suppose is what you trying to achieve is to check file existence in the scheduler container, so you have to do that check _in the bash script_, which is being built below.